Angular/my-app1/src/alert-button/alert-button.component.ts

```diff
@@ -0,0 +1,22 @@
+import { AfterViewInit, Component } from '@angular/core';
+
+@Component({
+  selector: 'app-alert-button',
+  standalone: true,
+  imports: [],
+  templateUrl: './alert-button.component.html',
+  styleUrl: './alert-button.component.scss'
+})
+export class AlertButtonComponent implements AfterViewInit {
+  ngAfterViewInit(): void {
+    const button = document.getElementById("alertButton")
+    if (button)
+    {
+      button.addEventListener("click",() =>
```

Comment:
No need of event listener. There is built in  "click" event for button which could call a function and do this action

Reply:
okay